### ISSUE TITLE
Error loading cif file

### ISSUE THREAD

**Issue by cjh1** [HEXRD/hexrd#94]
```
>>> from hexrd.material import Material
>>> mat = Material('nanopowder', 'nanopowder.cif')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/patrick/virtualenvs/hexrd/src/hexrd/hexrd/material.py", line 164, in __init__
    self._lparms, self.sgnum, self._atomtype, self._atominfo, self._U,
AttributeError: 'Material' object has no attribute '_U'
```

**Comment by psavery** [comment]
Note that this issue is part of PR #55

**Comment by psavery** [comment]
This is fixed in #55
